tribits/examples/TribitsExampleProject/README.md

```diff
@@ -20,12 +20,12 @@ After that configure pointing to some value TriBITS source dir <tribits-dir>
 ```
   cd BUILDS/SERIAL/ 
 
-  cmake -D \
+  cmake \
     -DTribitsExProj_TRIBITS_DIR=<tribits-dir> \
     -DTribitsExProj_ENABLE_TESTS=ON \
     -DTribitsExProj_ENABLE_ALL_PACKAGES=ON
     -DCMAKE_CXX_COMPILER=g++ \
-    ../../examples/TribitsExampleProject
```

Comment:
Actually, based on the assumed directory structure:

```
    ~/TribitsExampleProject.base/
        TribitsExampleProject/   # This directory
        BUILDS/
          SERIAL/
```

we are both wrong.  The correct path for CMake is:

```
cmake \
<...> \
../../TribitsExampleProject
```

The directory `BUILDS` is not **under** the source dir  `TribitsExampleProject`, it is **beside** it.

Reply:
Hello Ross,
I assumed: 
- one is currently in `TribitsExampleProject` (as it's marked # This directory) 
- one creates `BUILDS/SERIAL` dirs 
- then one (as it's given in the next step)  changes dir `cd BUILDS/SERIAL`
- and that's why `TribitsExampleProject` is in `../..`

Reply:
But anyway please feel free to close or edit this as you think would be the best :)

Reply:
@marcinwrobel1986, can you take look at the updated file in this PR and provide a review and signoff?

Reply:
> provide a review and signoff?

@marcinwrobel1986 so it looks like you can provide a review comment of your own PR but can't approve it (of course).